src/app/code/community/Wf/Bananastand/etc/system.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<config>
+  <tabs>
+    <banana translate="label" module="banana">
+      <label>Fera Ai</label>
+      <sort_order>100</sort_order>
+    </banana>
+  </tabs>
+  <sections>
+    <banana translate="label" module="banana">
+      <label>Fera Settings</label>
+      <tab>banana</tab>
+      <sort_order>1000</sort_order>
+      <show_in_default>1</show_in_default>
+      <show_in_website>1</show_in_website>
+      <show_in_store>1</show_in_store>
+
+      <groups>
+        <banana_group translate="label" module="banana">
+          <label>API Keys</label>
+          <frontend_type>text</frontend_type>
+          <sort_order>1000</sort_order>
+          <show_in_default>1</show_in_default>
+          <show_in_website>1</show_in_website>
+          <show_in_store>1</show_in_store>
+
+          <fields>
+            <public_key translate="label">
+              <label>Public Key: </label>
+              <frontend_type>obscure</frontend_type>
+              <sort_order>20</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>1</show_in_store>
+            </public_key>
+            <secret_key translate="label">
+              <label>Secret Key: </label>
+              <frontend_type>obscure</frontend_type>
+              <sort_order>90</sort_order>
+              <show_in_default>1</show_in_default>
+              <show_in_website>1</show_in_website>
+              <show_in_store>1</show_in_store>
+              <comment>You can find your API keys at https://app.fera.ai/store/settings</comment>
```

Comment:
`https://app.fera.ai/store/settings?tab=api` takes you directly there

Reply:
I originally tried this and noticed it didn't work. Digging a little deeper I see were using js and cookies to override the params[:tab] setting active. I'll modify this to use that url since it would work for people who never used the settings tab, but lmk if you want this behaviour modified.